CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+- Support for end-device clock correction through `AppTimeReq` and `AppTimeAns` as described in the LoRaWAN Application Layer Clock Synchronization Specification - TS003-2.0.0.
```

Comment:
I've added a changelog entry as part of the release in 84eda6ec79d2e3ada371b0864c81d4bdc89221ba . If you think that one suffices, let's drop this one from here.